src/main/java/seedu/address/model/SeplendidModelManager.java

```diff
@@ -449,8 +449,8 @@ public boolean hasNote(Note note) {
     }
 
     @Override
-    public void deleteNote(Note target) {
-        noteCatalogue.removeNote(target);
+    public Note deleteNote(int target) {
+        return noteCatalogue.removeNote(target);
```

Comment:
Would it be necessary to call the updateNoteList to update the note list to show to the user?

Reply:
I don't think it's required. During my testing, the list is updated accordingly when I delete note.

Reply:
Okay! I also just realised that after trying out.. Not sure why it doesn't work on some methods though. 👎